numbers_bernoulli.txt

```diff
@@ -0,0 +1 @@
+1
```

Comment:
hm, could you add in a few more data points, maybe just 0s or whatever. Single number makes it look trivial